scripts/sql/xavier-changes-2016_April/aatamsDB_queries_#268.sql

```diff
@@ -0,0 +1,70 @@
+﻿set search_path = aatams, public;
+
+begin;
+
+-- 1. Clean up animal_release table by deleting animal_release records with no corresponding surgery
+DELETE FROM animal_release ar
+WHERE NOT EXISTS (SELECT release_id FROM surgery s WHERE s.release_id = ar.id);
```

Comment:
> Should it not be the operator that corrects (and takes responsibility) for the mistake through the GUI?

In general I'd say yes but in this case those are obvious errors (we don't have any receiver in the northern hemisphere or West of Greenwich), @astecken agreed for us to fix those this way.

> How many records are affected?

About 250.